src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTagger.cs

```diff
@@ -44,13 +46,19 @@ internal sealed class InlineParameterNameHintsTagger : ITagger<IntraTextAdornmen
         private TextFormattingRunProperties? _format;
         private readonly IClassificationType _hintClassification;
         private readonly ForegroundThreadAffinitizedObject _threadAffinitizedObject;
+        private readonly IThreadingContext _threadingContext;
+        private readonly IToolTipService _toolTipService;
+        private readonly Lazy<IStreamingFindUsagesPresenter> _streamingFindUsagesPresenter;
```

Comment:
Let's try to keep this class sorted a bit more; let's have the services in one block, the buffer/view in another block, and then keep the cache and format map and format stuff by themselves like they were. Consider also breaking apart the constructor in the same way.